src/main/java/org/sagebionetworks/bridge/sdk/BridgeResearcherClient.java

```diff
@@ -258,15 +258,15 @@ public void deleteUploadSchemaByIdAndRevision(String schemaId, int revision) {
     }
 
     @Override
-    public UploadSchema getUploadSchemaById(String schemaId) {
+    public UploadSchema getUploadSchema(String schemaId) {
         session.checkSignedIn();
         checkArgument(isNotBlank(schemaId), Bridge.CANNOT_BE_BLANK, "schemaId");
         return get(config.getUploadSchemaByIdApi(schemaId), UploadSchema.class);
     }
 
     @Override
-    public ResourceList<UploadSchema> getUploadSchemaForStudy() {
+    public ResourceList<UploadSchema> getUploadSchemasAllRevisions() {
```

Comment:
I think this name is a little confusing. Unlike deleteUploadSchemaAllRevisions (which gets all revs of a specific schema ID), this one gets all schemas and all revisions for the entire study. The ForStudy() name is probably more accurate in this case, but I'm open to other suggestions.